README.rst

```diff
@@ -205,6 +205,17 @@ currently supported (these include ``customer.created``, ``customer.updated``,
 ``customer.subscription.created``, ``customer.subscription.deleted``
 ``invoice.created``, ``invoice.payment_succeeded``, ``invoice.payment_failed``).
 
+Flush stored data
+------------
+Flushing data programmatically can be useful
+if your are using localstripe with any test framework.
+
+Flushing stored data can be performed using the ``/_config`` route with DELETE http method:
```

Comment:
You mean ``/_config/store`` or ``/_config/reset``?

Reply:
/_config/store resource and DELETE request seems for me a good idea initially.

But because this project aim to be used for testing only, and "store" is more an internal keyword than something  straightly understandable by any new user I think the best one could of those routes:

* /_config/data + DELETE
* /_data + DELETE

What do you think about this ?